From 0caf02413c2bc85d8b56e3427fc79b100a89f089 Mon Sep 17 00:00:00 2001
From: Liam Howlett <liam.howlett@oracle.com>
Date: Tue, 25 Oct 2022 17:38:14 +0000
Subject: [PATCH 096/101] maple_tree: Fix mas_find_rev() comment

mas_find_rev() uses mas_prev_entry(), not mas_next_entry(), correct comment.

Signed-off-by: Liam R. Howlett <Liam.Howlett@oracle.com>
---
 lib/maple_tree.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index b1a342515..46d8c2d0e 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -6002,7 +6002,7 @@ void *mas_find(struct ma_state *mas, unsigned long max)
 	if (unlikely(!mas_searchable(mas)))
 		return NULL;
 
-	/* Retries on dead nodes handled by mas_next_entry */
+	/* Retries on dead nodes handled by mas_prev_entry */
 	return mas_next_entry(mas, max);
 }
 
-- 
2.38.1.143.g1fc3c0ad40

